clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/appgateway/model/AzureAppGatewayDescription.groovy

```diff
@@ -30,6 +30,7 @@ class AzureAppGatewayDescription extends AzureResourceOpsDescription {
   List<String> serverGroups
   List<AzureAppGatewayHealthcheckProbe> probes = []
   List<AzureAppGatewayRule> rules = []
+  List<AzureAppGatewayBackendAddressPool> backendAddressPools = []
```

Comment:
I'm not sure how we are going to use this list because it's not something we are going to bring it into the deck UI. Was this added for debugging purpose only? (in which case it makes total sense)